beacon-chain/p2p/handshake.go

```diff
@@ -174,7 +176,7 @@ func (s *Service) AddDisconnectionHandler(handler func(ctx context.Context, id p
 				s.peers.SetConnectionState(conn.RemotePeer(), peers.PeerDisconnected)
 				// Only log disconnections if we were fully connected.
 				if priorState == peers.PeerConnected {
-					log.WithField("activePeers", len(s.peers.Active())).Info("Peer Disconnected")
+					log.WithField("activePeers", len(s.peers.Active())).Info("Peer disconnected")
```

Comment:
change this back

Reply:
Hi my change was in response to https://github.com/prysmaticlabs/prysm/pull/6409#discussion_r446342716